cypress/integration/executions_spec.js

```diff
@@ -69,5 +69,42 @@ describe('Dashboard Executions Page', () => {
         });
       });
     });
+
+    it('should show a single execution', () => {
+      cy.contains('nav li a', 'Executions').as('executions');
+      cy.get('@executions').should('have.attr', 'href', '#/executions');
+      cy.get('@executions').click();
+
+      cy.url().should('include', 'executions');
+      cy.contains('.heading--xlarge', 'Executions');
+      const executionName = '50eaad71-bba8-4376-83d7-bb9cc1309b92';
+      const executionArn = 'arn:aws:states:us-east-1:596205514787:execution:TestSourceIntegrationIngestGranuleStateMachine-MOyI0myKEXzf:50eaad71-bba8-4376-83d7-bb9cc1309b92';
+      const stateMachine = 'arn:aws:states:us-east-1:596205514787:stateMachine:TestSourceIntegrationIngestGranuleStateMachine-MOyI0myKEXzf';
+      cy.get('table tbody tr td[class=table__main-asset]').within(() => {
+        cy.get(`a[title=${executionName}]`).click({force: true});
+      });
+
+      cy.contains('.heading--large', 'Execution');
+      cy.contains('.heading--medium', 'Visual workflow');
+
+      cy.get('.table--wrapper')
+        .within(() => {
+          cy.get('tr[data-value=31]').children('td').as('items');
+          cy.get('@items').eq(0).should('have.text', '31');
+          cy.get('@items').eq(1).should('have.text', 'ExecutionSucceeded');
+          cy.get('@items').eq(2).invoke('text').should('match', /\d{2}:\d{2}:\d{2} \d{2}\/\d{2}\/\d{2}$/);
+          cy.get('@items').eq(3).contains('More Details').click();
+          cy.get('@items').eq(3).contains('Less Details');
+        });
+
+      cy.get('.status--process')
+        .within(() => {
+          cy.contains('Execution Status:').next().should('have.text', 'Succeeded');
+          cy.contains('Execution Arn:').next().should('have.text', executionArn);
+          cy.contains('State Machine Arn:').next().should('have.text', stateMachine);
+          cy.contains('Started:').next().invoke('text').should('match', /\d{2}:\d{2}:\d{2} \d{2}\/\d{2}\/\d{2}$/);
+          cy.contains('Ended:').next().invoke('text').should('match', /\d{2}:\d{2}:\d{2} \d{2}\/\d{2}\/\d{2}$/);
+        });
+    });
```

Comment:
Is `View Execution Logs` tested?

Reply:
Added clicking on logs and a few tests on the logs page

Reply:
How hard would it be to get the contents of the "execution details"  and "execution logs" areas, parse the JSON string, and compare directly against the fixtures? Seems like it would be a very complete test with less steps, but I haven't thought about it deeply so let me know if this seems like a bad idea

Reply:
I think it would just as difficult to test everything directly with the fixture, because when the page is build it separate the data into different page elements, so we would still have to check each element. 
And on the other hand if we did just check that all the info was present on the overall page then that's not a very detailed test because we still want to make sure everything is where it is suppose to be.